### DISCUSSION TITLE
How to do VectorDB search (with filtering) using the new index lookup?

### DISCUSSION THREAD

**Discussion by KiranS-1952323** [microsoft/promptflow#3635] [Q&A]
I am building a flow where I need to query the Azure AI Search VectorDB. I have imported and vectorized the PDFs for my project with metadata, including clientID and projectID. Now, in my PromptFlow, I want to query the VectorDB and apply filters. This functionality is available in Azure AI Search but not in PromptFlow (or it is now deprecated). How can I do this?

I am trying to get the old vectorDB working, is this the right path?
![Screenshot_3](https://github.com/user-attachments/assets/7887e445-c5ae-4411-81d7-2d73d58d9f18)

**Comment by brynn-code** [comment]
@dans-msft @Adarsh-Ramanathan  could you please help take a look at this? Thank you.
